src/screens/Base.js

```diff
@@ -3,6 +3,7 @@ import { Router, routes } from '../route';
 import { Header, Footer } from '../components';
 import { HomeScreen } from '../screens';
 import { DetailsScreen } from './Details';
+import { FavoriteScreen } from './favorite';
```

Comment:
It should be `from './Favorite`, that's why pipeline failed :P

Reply:
done